instrumentation/spring/spring-boot-autoconfigure/src/testLogbackAppender/java/io/opentelemetry/instrumentation/spring/autoconfigure/instrumentation/logging/LogbackAppenderTest.java

```diff
@@ -57,15 +66,31 @@ void shouldInitializeAppender() {
     ConfigurableApplicationContext context = app.run();
     cleanup.deferCleanup(context);
 
-    LoggerFactory.getLogger("test").info("test log message");
+    MDC.put("key1", "val1");
+    MDC.put("key2", "val2");
+    try {
+      LoggerFactory.getLogger("test").info("test log message");
+    } finally {
+      MDC.clear();
+    }
 
-    assertThat(testing.logRecords())
+    List<LogRecordData> logRecords = testing.logRecords();
+    assertThat(logRecords)
         .satisfiesOnlyOnce(
             // OTel appender automatically added or from an XML file, it should not
             // be added a second time by LogbackAppenderApplicationListener
             logRecord -> {
               assertThat(logRecord.getInstrumentationScopeInfo().getName()).isEqualTo("test");
               assertThat(logRecord.getBody().asString()).contains("test log message");
+
+              Attributes attributes = logRecord.getAttributes();
+              assertThat(attributes.size())
+                  .isEqualTo(
+                      2); // key1 and key2, the code attributes should not be present because they
+              // are enabled in the logback.xml file but are disabled with a property
```

Comment:
same here, i'd suggest moving the multi-line comment above the line

Reply:
Done